### PR TITLE
style: add animations/transitions

### PR DESCRIPTION
**Pull request by pratikmdhr**
## WHAT IT DOES:

_Description of the pull request, what changed_

- Add fade-in transition to the Input/Result page components when the tabs on top are pressed
- Add transitions to the toggles (tabs on top)
- Added hover animation to the buttons
- Added focus state to the input fields

### WHAT'S LEFT:

_Delete if not necessary:_

- [ ] - Make it better (after the launch)

### CODE REVIEW

File: src/components/Input/InputField/WebsiteInputField.tsx [hunk 1]
```diff
@@ -30,8 +30,17 @@ const WebsiteInputField = ({
           name="website-url"
           value={inputUrl}
           placeholder="Enter a web page URL"
+          pattern="^(https?|ftp):\/\/[^\s/:?#][^\s]*$" // Regex to check if the input is a valid URL
           required
-          onChange={(e) => setInputUrl(e.target.value)}
+          onChange={(e) => {
+            // This will display the error message on browser's default tooltip
+            if (!e.target.validity.valid) {
+              e.target.setCustomValidity("Please enter a valid URL");
```
Comment (MaiaS) [inline src/components/Input/InputField/WebsiteInputField.tsx]: Oh didn't know we could do that. We should probably use the browser's validity tooltips to be more consistent :). I can edit my PR tomorrow to use the default tooltips

File: src/components/Result/Result.tsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ import SongResult from "./ResultContent/SongResult";
 import TextResult from "./ResultContent/TextResult";
 import ResultPageContentToggler from "../utility-components/result/ResultPageContentToggler";
 import OriginalContent from "./ResultContent/OriginalContent";
+import { ContentType } from "~/types";
```
Comment (MaiaS) [inline src/components/Result/Result.tsx]: unused import :)

**Review by MaiaS — APPROVED** [review]
Looks great! Thank you for doing this!

File: src/components/Result/Result.tsx (hunk 2 above)
Comment (pratikmdhr) [reply src/components/Result/Result.tsx]: oops

**Pull request merged**